### ISSUE TITLE
actionable tag

### ISSUE THREAD

**Issue by dvribeira** [CARA-ch/ch-emed-epr#81]
During the IHE+HL7 MPD calls the issue was raised about the use of the actionable meta tag. In principle medication requests that are supposed to be acted upon (in our case, any request conveyed by a PRE or PADV) should contain the actionable tag while other requests, conveyed for information purposes (e.g. a patient summary) should not contain it. It seems there is some confusion regarding this tag and that it is not widely use it (during the call it seemed only Dutch implementations were using it). The issue will be raised to HL7 to ask for guidance and whether this should be reviewed.

It might be that we should have these tags present for PRE and PADV requests. I am unsure whether they should be present within PML requests.

**Comment by qligier** [comment]
The 'actionable' intent is also conveyed in the MedicationRequest.status:

> 	active: The prescription is 'actionable', but not all actions that are implied by it have occurred yet.

**Comment by dvribeira** [comment]
Yes, that's why it is unclear whether this would be needed/desired at all. I will wait to see what's the feedback on that from HL7 and the discussion on MPD. The Dutch are using it to differentiate between requests sent to pharmacies to be acted upon and request resources sent for information purposes (while they are indeed active requests).
